dev-tools/iota-indexer-monitoring/README.md

```diff
@@ -0,0 +1,16 @@
+# Prometheus and Grafana monitoring for `iota-indexer`
+
+This docker-compose configuration allows launching instances of the Prometheus and Grafana applications for monitoring of the deployed `iota-indexer` instance.
+
+## Prerequisites
+
+In order to run this monitoring setup, you first need to have `iota-indexer` setup running. By default, Prometheus will listen at the `client-metrics-port` at `9184` in order to scrape the metrics.
```

Comment:
Nit: Maybe we could emphasise that in order to `iota-indexer` to correctly expose its metrics it should be in the same docker network as the prometheus and grafana services.

Reply:
Fixed by: https://github.com/iotaledger/iota/pull/6378/commits/4fe2ab2362d535f3924417cb023c7fd6eddb2969